cpp/src/binaryop/binaryop.cpp

```diff
@@ -144,34 +148,96 @@ void scalar_col_valid_mask_and(gdf_size_type& out_null_count,
 
 namespace jit {
 
+  const std::string hash = "prog_binop";
+
+  const std::vector<std::string> compiler_flags { "-std=c++14" };
+  const std::vector<std::string> headers_name
+        { "operation.h" , "traits.h", cudf_types_h };
+  
+  std::istream* headers_code(std::string filename, std::iostream& stream) {
```

Comment:
Documentation.

Reply:
Here `src/jit/launcher` has the documentation for the `launcher` class.